utils/mount/overlay2.go

```diff
@@ -46,15 +45,15 @@ func NewMountDriver() Interface {
 	return &Default{}
 }
 
-func supportsOverlay() bool {
-	if err := exec.Command("modprobe", "overlay").Run(); err != nil {
-		return false
-	}
+func supportsOverlay() (flag bool) {
 	f, err := os.Open("/proc/filesystems")
 	if err != nil {
 		return false
 	}
 	defer func() {
+		if !flag {
+			logrus.Infof("The current environment does not support overlay")
```

Comment:
Maybe warn would be better

Reply:
Yep, Thx!

Reply:
> I'm considering if `modprobe overlay` failed, why do we allow the process to keep on. Why no just throw the error? I'm afraid that relaxing the restriction will incur more problems. And I think `modprobe overlay` is enough for now. If there are users need to relax the restriction. We can provide a env or option for user. @YTGhost @ningpengtao-coder

I think using `/proc/filesystems` to detect supported filesystems seems to be sufficient in a linux environment? In wsl and container environments, `modprobe overlay` may not be supported for use.